build/common-config.sh

```diff
@@ -157,6 +157,7 @@ var_set() {
 	}
 
 do_export() {
+        export PDO_SOURCE_ROOT
```

Comment:
i think it would be good if common-config.sh sets PDO_SOURCE_ROOT to the PDO root folder of common-config.sh. Default values do not make any sense, IMHO. (the file does not exist until you clone PDO repo to SRC_ROOT, at which point you already have an implicit SRC_ROOT, so you are doing discovery of what is already present)

Reply:
For "default value", I mean the value it has right now, namely: `PDO_SOURCE_ROOT="$(realpath ${SCRIPTDIR}/..)"`.

So, just as with do with other variables, we would have something like:
`env_val[PDO_SOURCE_ROOT]="${PDO_SOURCE_ROOT:-$(realpath ${SCRIPTDIR}/..)"`

Reply:
I don't believe there is any value in making it configurable. You're pulling it out of a source file in the source directory.

Reply:
@prakashngit that is exactly what it is set to. look at the top of the file.